lib/wanda/execution/evaluation.ex

```diff
@@ -224,4 +237,51 @@ defmodule Wanda.Execution.Evaluation do
   defp result_weight(:critical), do: 2
   defp result_weight(:warning), do: 1
   defp result_weight(:passing), do: 0
+
+  defp compute_values(values, facts, env) do
+    Enum.map(values, &value_from_conditions(&1, facts, env))
+  end
+
+  defp value_from_conditions(
+         %Wanda.Catalog.Value{
+           name: name,
+           default: default,
+           conditions: conditions
+         },
+         facts,
+         env
+       ) do
+    scoped_facts = map_facts_to_evaluation_scope(facts)
+
+    computed_value =
+      Enum.find_value(conditions, default, fn %Wanda.Catalog.Condition{
+                                                value: value,
+                                                expression: expression
+                                              } ->
+        case Abacus.eval(expression, %{"facts" => scoped_facts, "env" => env}) do
+          {:ok, true} ->
+            value
+
+          _ ->
+            false
+        end
+      end)
+
+    %Wanda.Execution.Value{
+      name: name,
+      value: computed_value
+    }
+  end
+
+  defp map_facts_to_evaluation_scope(facts) do
+    Enum.reduce(facts, %{}, fn %Fact{name: name, value: value}, acc ->
+      put_in(acc, [name], value)
+    end)
+  end
+
+  defp map_values_to_evaluation_scope(values) do
+    Enum.reduce(values, %{}, fn %Wanda.Execution.Value{name: name, value: value}, acc ->
+      put_in(acc, [name], value)
```

Comment:
not sure you need put_in here, this is used usually for nested maps. 
We are trying to build a map where the name of the execution/fact is the key and value is the value.

```elixir 
Enum.into(%{}, fn %{name: name, value: value} -> {name, value} end)
```

Also, consider that those two functions are the same basically (since both fact and value have name and value fields) so they might be redundant.